src/3d/chunks/qgschunkedentity_p.cpp

```diff
@@ -411,6 +415,8 @@ QgsChunkQueueJob *QgsChunkedEntity::startJob( QgsChunkNode *node )
     QgsEventTracing::addEvent( QgsEventTracing::AsyncBegin, QStringLiteral( "3D" ), QStringLiteral( "Load " ) + node->tileId().text(), node->tileId().text() );
 
     QgsChunkLoader *loader = mChunkLoaderFactory->createChunkLoader( node );
+    if ( !loader )
+      return nullptr;
```

Comment:
Is this addition necessary?

The returned loader must be valid - otherwise things can break elsewhere (we can get to invalid state in the state machine and loading queue). If I understand correctly, with your other additions, startJob() should not even get called when the entity is not valid, so this check should not be needed...